andela/ah-technocrats
# #161966619  user should be able to see the the list and profile of other users

### What does this pull request do?
 Gets a list of all existing authors profile 
### Description of the tasks to be completed?
- add functionality to enable authenticated users to get a list of all existing author profile
### How should this be manually tested?
- clone the repo `git clone https://github.com/andela/ah-technocrats.git`
- `cd ah-technocrats`
- checkout out to this branch `git checkout ft-list-users-161966619 `
- make virtual environment `virtualenv venv`
- activate the virtual environment `source venv/bin/activate`
- install all the dependencies from the requirements file ` pip install -r requirements.txt`
- make migrations  `python manage.py makemigrations`
- apply the migrations `python manage.py migrate`
- run the server  `python manage.py runserver`
- go to Postman to confirm the functionality.
### Any background context you want to provide?
- Once a user logs in, they can access be able to get the list and profiles of existing authors.
### What are the relevant Pivotal Tracker Stories
[#161966619](https://www.pivotaltracker.com/story/show/161966609)
# Checklist:


<img width="1440" alt="screen shot 2018-12-07 at 10 33 40" src="https://user-images.githubusercontent.com/36000749/49633763-a4b96d80-fa0b-11e8-8e0a-885aafa07f74.png">
